compose/cli/main.py

```diff
@@ -1392,10 +1398,13 @@ def log_printer_from_project(
         log_args=log_args)
 
 
-def filter_containers_to_service_names(containers, service_names):
+def filter_attached_containers(containers, service_names, attach_dependencies):
     if not service_names:
         return containers
 
+    if attach_dependencies:
```

Comment:
Since the content of this `if` is equal to the preceding one, please, just add an `or` to the first one

Reply:
Sure - I've done that.